app/alerts.py

```diff
@@ -99,18 +111,19 @@ def build_sites_markers(dpt_code=None):
 
     # We build a list of markers containing the info of each site/camera
     markers = []
-    for i, row in camera_positions.iterrows():
-        lat = row['Latitude']
-        lon = row['Longitude']
-        site_name = row['Tours']
-        nb_device = row['Nombres Devices']
-        markers.append(dl.Marker(id=f'site_{i}',    # Necessary to set an id for each marker to receive callbacks
+    for row in camera_positions:
+        id = row['id']
+        lat = row['lat']
+        lon = row['lon']
+        site_name = row['name']
+        # nb_device = row['Nombres Devices']
+        markers.append(dl.Marker(id=f'site_{id}',    # Necessary to set an id for each marker to receive callbacks
```

Comment:
Just to mention that if we do make the change from `id` to `site_id` (mentioned in the comment right above), this line will have to be edited as well.